subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -450,26 +469,125 @@ static int nct_provision(void)
 }
 
 #if defined(CONFIG_AWS_FOTA)
+static int job_status_stream(const struct nct_dc_data *dc_data)
+{
+	if (dc_data == NULL) {
+		return -EINVAL;
+	}
+
+	if (nct.job_status_endp.utf8 == NULL) {
+		LOG_ERR("Job status topic not set");
+		return -EACCES;
+	}
+
+	struct mqtt_publish_param publish = { .message.topic.qos =
```

Comment:
IMO it would look nicer if you moved the entire `.message.topic.qos = MQTT_QOS_0_AT_MOST_ONCE` line below.